frontend/src/pages/PlannerWeightsPage.module.scss

```
@@ -2,20 +2,19 @@
 
 .weightsHeader {
   @extend .layout-row;
-  margin-left: 25px;
+  margin: 0 25px;
   align-items: flex-start;
 
   .title {
     @extend .layout-row;
     gap: 12px;
-    align-items: center;
+    align-items: flex-start;
```

Noticed we can get rid of the extra div around the infoTooltip by setting the align as baseline
```suggestion
    align-items: baseline;
```